ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/AbstractCallOperation.java

```diff
@@ -199,7 +199,7 @@ public OperationResult execute(final MessageFrame frame, final EVM evm) {
               .sender(sender(frame))
               .value(value(frame))
               .apparentValue(apparentValue(frame))
-              .code(new Code(contract != null ? contract.getCode() : Bytes.EMPTY))
+              .code(frame.getWorldState().getContract(contract).orElse(new Code()))
```

Comment:
So what happens if I try to get code on a EOA?

Reply:
you get a new Code() with nothing in it. I suppose we could also put Bytes.EMPTY in the new Code() for consistency with previous, if that's what you're getting at.

Reply:
From your code changes it looks like you’d get an IllegalArgumentException.

Reply:
sorry, I don't see it. which part would throw that?

Reply:
perhaps a static Code.EMPTY constant would make the behavior more clear instead of a zero arg constructor.

Reply:
I was referring to `getContract` hitting the cache, and I erroneously thought it would the IllegalArgumentException thrown in the CacheLoader class.